csunplugged/tests/resources/ResourcesTestDataGenerator.py

```diff
@@ -0,0 +1,29 @@
+"""Create test data for resource tests."""
+
+from resources.models import Resource
+
+
+class ResourcesTestDataGenerator:
+    """Class for generating test data for resource tests."""
+
+    def create_test_resource(self, slug, name, webpage_template, generation_view):
+        """Create resource object.
+
+        Args:
+            slug: String of resource slug.
+            name: String of resource name.
+            webpage_template: String describing path to HTML form template.
+            generation_view: String of Python view module filename.
```

Comment:
(maybe I'm wrong) but didn't we decided to give the type using python types? e.g. `resource slug (str).` ?